### PR TITLE
[SECURESIGN-663] Add option to set external ctlog service. Refactor services port name.

### PR DESCRIPTION
**Pull request by fghanmi**
- Add the option to use an external CTLog
- Expose Trillian log server via a route
- Refactor of services port name done on: [Ctlog, Trillian]
Another PR will be created for the rest of components.

### CODE REVIEW

File: internal/controller/common/utils/kubernetes/service.go [hunk 1]
```diff
@@ -12,7 +12,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
-func CreateService(namespace string, name string, port int, labels map[string]string) *corev1.Service {
+func CreateService(namespace string, name string, portName string, port int, labels map[string]string) *corev1.Service {
```
Comment (osmman) [inline internal/controller/common/utils/kubernetes/service.go:15]: Please create a separate PR for refactoring `CreateService` method and it's required changes.

**Review by osmman — CHANGES_REQUESTED** [review]

**Comment by fghanmi** [comment]
To be declined: 
Will create sub PRs instead.

**Pull request closed without merging**